timesketch/api/v1/resources/settings.py

```diff
@@ -27,46 +15,78 @@ class SystemSettingsResource(Resource):
     @login_required
     def get(self):
         """GET system settings.
-
         Returns:
             JSON object with system settings.
         """
-        # Settings from timesketch.conf to expose to the frontend clients.
-        settings_to_return = ["DFIQ_ENABLED"]
-        result = {}
-
-        for setting in settings_to_return:
-            result[setting] = current_app.config.get(setting)
```

Comment:
Why replace the old iterative way to do this? The benefit was to just add the setting name into the list and it would be exposed to the frontend.

Reply:
Fixed